nixos/tests/nfs/kerberos.nix

```diff
@@ -71,7 +71,7 @@ in
         services.nfs.server.createMountPoints = true;
         services.nfs.server.exports =
           ''
-            /data *(rw,no_root_squash,fsid=0,sec=krb5p)
+            /data *(rw,no_root_squash,fsid=1,sec=krb5p)
```

Comment:
Interesting. I was using an fsid: https://github.com/TUM-DSE/doctor-cluster-config/blob/e88b4edcdbeb0701a4606783cce083e8e9572fde/modules/nfs/server.nix#L41 but still saw breakages.

Reply:
Ah. It's actually not using fsid=0 that breaks it?

Reply:
Yeah, this "fix" is making the test fail. It currently passes even though NFS4 is broken for many users.